CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - `t:EdgeDB.Object.t/0` to be `opaque`.
+- injection of implicit `:id` field into objects.
```

Comment:
This sounds like ID injection was added.

Reply:
changed to make it more expressive